include/avif/avif.h

```diff
@@ -289,7 +289,10 @@ enum
     AVIF_MATRIX_COEFFICIENTS_SMPTE2085 = 11,
     AVIF_MATRIX_COEFFICIENTS_CHROMA_DERIVED_NCL = 12,
     AVIF_MATRIX_COEFFICIENTS_CHROMA_DERIVED_CL = 13,
-    AVIF_MATRIX_COEFFICIENTS_ICTCP = 14
+    AVIF_MATRIX_COEFFICIENTS_ICTCP = 14,
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+    AVIF_MATRIX_COEFFICIENTS_YCGCO_R = 15,
```

Comment:
Should this be named `AVIF_MATRIX_COEFFICIENTS_YCGCO_R`, because it is called YCgCo-Ro in the proposed change to the standard?

Reply:
Indeed: I added support for RE and RO